.mergify.yml

```
@@ -10,6 +10,17 @@ pull_request_rules:
         strict: smart
       label:
         add: ["auto merged"]
+  - name: Automatically merge PRs from maintainers on CI success and review
```

If we do decide to keep this change, the description should be accurate
```suggestion
  - name: Automatically merge PRs from maintainers on CI success
```